orga/app/routes/authenticated/students/list.js

```diff
@@ -1,11 +1,35 @@
 import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
 import Route from '@ember/routing/route';
 
 export default class ListRoute extends Route {
 
+  queryParams = {
+    lastName: {
+      refreshModel: true
+    },
+  };
+
   @service currentUser;
 
-  model() {
-    return this.currentUser.organization.students;
+  model(params) {
+    return this.store.query('student', {
+      filter: {
+        organizationId: this.currentUser.organization.id,
+        lastName: params.lastName,
+      },
+    }, { reload: true });
```

Comment:
```suggestion
    });
```
Je ne suis pas sûre que le reload soit nécéssaire